README.md

```diff
@@ -1 +1,68 @@
-# workflow-image-scan
\ No newline at end of file
+# Reusable GitHub Workflow to Scan Images
+
+> This workflow is highly specific for a few Portworx-internal repositories - may not be suited for generic usage.
+
+The main goal this workflow is to:
+
+1. call Lacework scanner action ([lacework/lw-scanner-action](https://github.com/lacework/lw-scanner-action)) for the
+   input images,
+2. builds a scan result summary section and adds it to the end of the Actions run page,
+3. adds a comment into the pull request containing the scan results in short form.
```

Comment:
nit: I'd capitalize each point and end with periods instead of commas (I get that it's like a list, but it looks a bit weird imho).
And make the sentences make sense as the second part of "`The main goal of this workflow is to:`".

```suggestion
1. Call Lacework scanner action ([lacework/lw-scanner-action](https://github.com/lacework/lw-scanner-action)) for the
   input images.
2. Build a scan result summary section and add it to the end of the Actions run page.
3. Add a comment into the pull request containing the scan results in short form.
```